fluent-react/src/provider.ts

```diff
@@ -66,13 +60,14 @@ function isIterable(
     );
   }
 
-  if (Symbol.iterator in Object(prop)) {
+  if (prop instanceof ReactLocalization) {
     return null;
   }
 
   return new Error(
-    `The ${propName} prop supplied to ${componentName} must be an iterable.`
+    `The ${propName} prop supplied to ${componentName} must be an instance \
+of ReactLocalization.`
   );
 }
 
-export let MemoLocalizationProvider = memo(LocalizationProvider);
+export const MemoLocalizationProvider = memo(LocalizationProvider);
```

Comment:
I kept `memo` here for now, but I think it's not needed. If I'm reading the code right, [`Context.Provider` is already memoized](https://github.com/facebook/react/blob/9d67847f7b41e408016cfa16450323b3fee8bc56/packages/react-reconciler/src/ReactFiberBeginWork.js#L2694), effectively.